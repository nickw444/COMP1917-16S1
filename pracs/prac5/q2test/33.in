33
9 4 -4 -5 4 -10 0 4 -2 9 -8 -5 10 -5 9 -4 9 -3 -9 -9 -10 5 -8 0 7 7 -1 -4 -3 8 -5 6 2
