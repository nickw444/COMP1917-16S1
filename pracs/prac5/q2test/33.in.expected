average of array: -0.060606
